app/controllers/account/projects_controller.rb

```diff
@@ -1,23 +1,24 @@
 class Account::ProjectsController < Account::AccountController
+	before_action :set_workspace
   before_action :set_project, only: [:show, :edit, :update, :destroy]
 
   def index
-    @projects = current_user.projects.order(created_at: :desc)
+    @projects = @workspace.projects.order(created_at: :desc)
```

Comment:
add ordered scope to Project model

Reply:
In addition, you don't have an index for created_at column, that means sorting will be performed slowly.
In most cases, it is better to sort by id instead of created_at